.gitlab-ci.yml

```diff
@@ -282,12 +278,14 @@ pkg:opam:
   dependencies: []
   script:
     - set -e
-    - opam pin add coq .
-    - opam pin add coqide-server .
-    - opam pin add coqide .
+    - opam pin add --kind=path coq.$COQ_VERSION .
+    - opam pin add --kind=path coqide-server.$COQ_VERSION .
+    - opam pin add --kind=path coqide.$COQ_VERSION .
```

Comment:
How's this related?

Reply:
Oh I forgot to update the PR message, the commit message has the info.

Reply:
@Zimmi48 proposes here to use `dev` as `COQ_VERSION` ; I'm unsure if that would create problem, semantics of `pin` and `dev` are very tricky. Does anyone know?